custom/icds/data_management/doc_processors.py

```diff
@@ -0,0 +1,121 @@
+from datetime import date
+from xml.etree import cElementTree as ElementTree
+
+from casexml.apps.case.mock import CaseBlock
+
+from corehq.apps.hqcase.utils import (
+    get_last_non_blank_value,
+    submit_case_blocks,
+)
+from corehq.apps.users.util import SYSTEM_USER_ID
+from corehq.form_processor.backends.sql.dbaccessors import CaseAccessorSQL
+from corehq.form_processor.exceptions import CaseNotFound
+from corehq.form_processor.interfaces.dbaccessors import CaseAccessors
+from corehq.util.doc_processor.interface import BaseDocProcessor
+from custom.icds.utils.location import find_test_awc_location_ids
+
+DOB_PROPERTY = "dob"
+MOTHER_NAME_PROPERTY = "mother_name"
+CUT_OFF_AGE_IN_YEARS = 6
+MOTHER_INDEX_IDENTIFIER = "mother"
+PHONE_NUMBER_PROPERTY = "contact_phone_number"
+HAS_MOBILE_PROPERTY = "has_mobile"
+HAS_MOBILE_PROPERTY_NO_VALUE = "no"
+
+
+class DataManagementDocProcessor(BaseDocProcessor):
+    def __init__(self, domain):
+        self.domain = domain
+
+    @staticmethod
+    def _create_case_blocks(updates, case_property):
+        case_blocks = []
+        for case_id, new_value in updates.items():
+            case_block = CaseBlock(case_id,
+                                   update={case_property: new_value},
+                                   user_id=SYSTEM_USER_ID)
+            case_block = ElementTree.tostring(case_block.as_xml()).decode('utf-8')
+            case_blocks.append(case_block)
+        return case_blocks
+
+    def handle_skip(self, doc):
+        print('Unable to process case {}'.format(doc['_id']))
```

Comment:
who will see this message?

Reply:
Currently just logged for testing, would be enhanced to share these case ids with the user initiating the request when we take up feedback to user in a follow up PR.